broker/config/crds/deployment.servicefabrik.io_v1alpha1_virtualhosts.yaml

```diff
@@ -37,6 +37,8 @@ spec:
               type: string
             state:
               type: string
+            appliedOptions:
+              type: string
```

Comment:
@subhankarc is this modification in vhost CRD fine or is there any other CRD I need to modify? @pranjaljain

Reply:
looks fine.